core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/SequenceActions.scala

```diff
@@ -312,13 +317,15 @@ protected[actions] trait SequenceActions {
    * The method distinguishes between invoking a sequence or an atomic action.
    * @param user the user executing the sequence
    * @param futureAction the future which fetches the action to be invoked from the db
+   * @param originAction the invoked fully qualified action name that not rewritten
    * @param accounting the state of the sequence activation, contains the dynamic activation count, logs and payload for the next action
    * @param cause the activation id of the first sequence containing this activations
    * @return a future which resolves with updated accounting for a sequence, including the last result, duration, and activation ids
    */
   private def invokeNextAction(
     user: Identity,
     futureAction: Future[WhiskActionMetaData],
+    originAction: FullyQualifiedEntityName,
```

Comment:
Is this being used in the call tree?

Reply:
It seems not using in the call tree. I'll fix it.